src/containers/NonBitGoRecoveryCoin/NonBitGoRecoveryCoin.tsx

```diff
@@ -743,9 +748,12 @@ function Form() {
 
               await window.commands.writeFile(
                 showSaveDialogData.filePath,
-                JSON.stringify(recoverData,
-                  (key, value) => typeof value === 'bigint' ? value.toString() : value,
-                  2),
+                JSON.stringify(
+                  recoverData,
+                  (key, value) =>
+                    typeof value === 'bigint' ? value.toString() : value,
+                  2
+                ),
```

Comment:
this serializer should exist as a common utility

Reply:
The contents of this file where unchanged outside of changing the order for the params to `setBitGoEnvironment`. The other changes are whitespace/format changes that happened via my editor.